viadot/sources/mediatool.py

```diff
@@ -0,0 +1,302 @@
+        dfs = []
+        for id in vehicle_ids:
+            url = f"https://api.mediatool.com/vehicles/{id}"
+
+            try:
+                response = handle_api_response(
+                    url=url,
+                    headers=self.header,
+                    method="GET",
+                )
+            except Exception:
+                raise NameError(f"No vehicle with id {id} was found")
+
+            response_dict = json.loads(response.text)
+            df_single = pd.DataFrame(response_dict["vehicle"], index=[0])
+            dfs.append(df_single)
+
+        if return_dataframe is True:
+            df = pd.concat(dfs)
+            function_name = inspect.stack()[0][3]
+            df_updated = self.rename_columns(df=df, column_suffix=function_name)
+            return df_updated
+
+        return response_dict["vehicles"]
+
+    def get_organizations(
+        self, user_id: str = None, return_dataframe: bool = True
+    ) -> pd.DataFrame:
+        """
+        Get organizations data based on the user ID. Returns DF or dict.
+
+        Args:
+            user_id (str): User ID.
+            return_dataframe (bool, optional): Return a dataframe if True. If set to False, return the data as a dict.
+            Defaults to True.
+
+        Returns:
+            pd.DataFrame: Default return dataframe. If 'return_daframe=False' then return list of dicts.
+        """
+        user_id = user_id or self.user_id
+        url_organizations = f"https://api.mediatool.com/users/{user_id}/organizations"
+
+        response = handle_api_response(
+            url=url_organizations,
+            headers=self.header,
+            method="GET",
+        )
+        response_dict = json.loads(response.text)
+        organizations = response_dict["organizations"]
+
+        list_organizations = []
+        for org in organizations:
+            try:
+                list_organizations.append(
+                    {
+                        "_id": org["_id"],
+                        "name": org["name"],
+                        "abbreviation": org["abbreviation"],
+                    }
+                )
+            except KeyError:
+                list_organizations.append(
+                    {
+                        "_id": org["_id"],
+                        "name": org["name"],
+                        "abbreviation": None,
+                    }
+                )
+
+        if return_dataframe is True:
+            df = pd.DataFrame.from_dict(list_organizations)
+            function_name = inspect.stack()[0][3]
+            df_updated = self.rename_columns(df=df, column_suffix=function_name)
+            return df_updated
+
+        return list_organizations
+
+    def get_media_types(
+        self, media_type_ids: List[str], return_dataframe: bool = True
+    ) -> pd.DataFrame:
+        """
+        Get media types data based on the media types ID. User have to provide list of media type IDs.
+        Returns DF or dict.
```

Comment:
maybe its better to say "DataFrame or Dict"? you are specifying it in a few places